objectbox/test/query_test.dart

```diff
@@ -361,208 +364,39 @@ void main() {
     expect(remaining.map((e) => e.id), equals([1, 4]));
   });
 
-  test('.count items after grouping with and/or', () {
```

Comment:
A bunch of unrelated query tests got removed, was this intentional?

Reply:
nope, restored, thanks